script/hassfest/manifest.py

```diff
@@ -126,14 +126,15 @@ class QualityScale(IntEnum):
 ]
 
 
-def documentation_url(value: str) -> str:
+def own_documentation_url(value: str) -> str:
     """Validate that a documentation url has the correct path and domain."""
     if value in DOCUMENTATION_URL_EXCEPTIONS:
         return value
 
     parsed_url = urlparse(value)
-    if parsed_url.scheme != DOCUMENTATION_URL_SCHEMA:
-        raise vol.Invalid("Documentation url is not prefixed with https")
+    documentation_url(value)
```

Comment:
I think you can use `parsed_url = documentation_url(value)` and remove the line above

Reply:
But then I get a str, not a parsed url. I could also always just do a `.startswith("https://www.home-assistant.io/integrations")`